File: 프로그래머스/unrated/181930. 주사위 게임 2/주사위 게임 2.py
def solution(a, b, c):
    data = int(a == b) + int(a == c) + int(b == c)
    print(data)
    if data == 1:
        return (a + b + c) * (a**2 + b**2 + c**2)
    elif data == 3:
        return (a + b + c) * (a**2 + b**2 + c**2 ) * (a**3 + b**3 + c**3 )
    else:
        return a+b+c

File: 프로그래머스/0/181930. 주사위 게임 2/주사위 게임 2.java
class Solution {
    public int solution(int a, int b, int c) {
        if (a == b && b == c) {
            return (a + b + c) * (a * a + b * b + c * c) * (a * a * a + b * b * b + c * c * c);
        }
        if (a == b || b == c || a == c) {
            return (a + b + c) * (a * a + b * b + c * c);            
        }
        return a + b + c;
    }
}